src/test/ui/binding/expr-match-generic-unique1.rs

```diff
@@ -2,7 +2,7 @@
 
 fn test_generic<T: Clone, F>(expected: Box<T>, eq: F) where F: FnOnce(Box<T>, Box<T>) -> bool {
     let actual: Box<T> = match true {
-        true => { expected.clone() },
+        true => expected.clone(),
```

Comment:
I think you should skip changing tests. It may unintentionally change the meaning of the tests. We don't run rustfmt on tests at all.